### PR TITLE
[feat]tools-v2: add bs query volume clone-recover

### PR DESCRIPTION
**Pull request by XR-stb**
### What problem does this PR solve?

Issue Number: #2584 

Problem Summary:

### What is changed and how it works?

What's Changed:

use a common query clone and recover status

How it Works:

abstract public code into functions

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by XR-stb** [comment]
cicheck

**Review by Cyber-SiKu — COMMENTED** [review]
Add some instructions to the readme

File: tools-v2/pkg/cli/command/curvebs/query/volume/clone_recover/clone_recover.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+		var resp struct {
+			Code       string              `json:"Code"`
+			TaskInfos  []map[string]string `json:"TaskInfos"`
+			TotalCount int                 `json:"TotalCount"`
+		}
+		if err := json.Unmarshal([]byte(result), &resp); err != nil {
+			return nil, err
+		}
+		if resp.Code != "0" {
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/query/volume/clone_recover/clone_recover.go:77]: defined as constant

**Review by Cyber-SiKu — COMMENTED** [review]
Supplement the output

**Pull request closed without merging**